src/acl/ServerCertificate.cc

```diff
@@ -24,7 +24,10 @@ ACLServerCertificateStrategy::match(ACLData<MatchType> * &data, ACLFilledCheckli
     Security::CertPointer cert;
     if (checklist->serverCert)
         cert = checklist->serverCert;
```

Comment:
Please add a comment to the ACLFilledChecklist::serverCert declaration:

```C++
    /// Peer certificate being checked by ssl_verify_cb(). In other contexts,
    /// the peer certificate is retrieved via ALE or ConnStateData::serverBump.
    Security::CertPointer serverCert;
```

Reply:
I use a similar but not the same comment commit ad5b76e9b1